client/src/components/App/index.js

```diff
@@ -1,44 +1,46 @@
-          <Route exact path="/signup" component={SignUp} />
-          {/* http://localhost:3000/login */}
-          <Route exact path="/login" component={Login} />
-          {/* http://localhost:3000/logout */}
-          {/* <Route exact path='/login' component={ Logout }/> */}
-          {/* http://localhost:3000/users/me */}
-          {/* <Route exact path='/users/me' component={ UserProfile }/> */}
-          {/* http://localhost:3000/reservation/pre/:screeningId */}
-          {/* <Route path='/reservation/pre/:screeningId' component={ ReservationPreview }/> */}
-          {/* http://localhost:3000/reservation/chooseSeats/:screeningId */}
-          {/* <Route path='/reservation/chooseSeats/:screeningId' component={ ReservationDetails }/> */}
-          {/* http://localhost:3000/reservation/details/:reservationId */}
-          {/* <Route path='/reservation/details/:reservationId' component={ ReservationConfirmation }/> */}
-          {/* http://localhost:3000/reservation/payment/:reservationId */}
-          {/* <Route path='/reservation/payment/:reservationId' component={ ReservationPayment }/> */}
-          {/* http://localhost:3000/reservation/confirm/:reservationId */}
-          {/* <Route path='/reservation/confirm/:reservationId' component={ ReservationConfirmation }/> */}
+      <Fragment>
+        <Navbar />
+        <div className="App">
+          <Route exact path="/" component={Landing} />
+          <Switch>
+            {/* http://localhost:3000/movies */}
+            <Route exact path="/movies" component={MovieList} />
+            {/* http://localhost:3000/signup */}
+            <Route exact path="/signup" component={SignUp} />
+            {/* http://localhost:3000/login */}
+            <Route exact path="/login" component={Login} />
+            {/* http://localhost:3000/logout */}
+            {/* <Route exact path='/login' component={ Logout }/> */}
+            {/* http://localhost:3000/users/me */}
+            {/* <Route exact path='/users/me' component={ UserProfile }/> */}
+            {/* http://localhost:3000/reservation/pre/:screeningId */}
+            {/* <Route path='/reservation/pre/:screeningId' component={ ReservationPreview }/> */}
+            {/* http://localhost:3000/reservation/chooseSeats/:screeningId */}
+            {/* <Route path='/reservation/chooseSeats/:screeningId' component={ ReservationDetails }/> */}
+            {/* http://localhost:3000/reservation/details/:reservationId */}
+            {/* <Route path='/reservation/details/:reservationId' component={ ReservationConfirmation }/> */}
+            {/* http://localhost:3000/reservation/payment/:reservationId */}
+            {/* <Route path='/reservation/payment/:reservationId' component={ ReservationPayment }/> */}
+            {/* http://localhost:3000/reservation/confirm/:reservationId */}
+            {/* <Route path='/reservation/confirm/:reservationId' component={ ReservationConfirmation }/> */}
```

Comment:
I don't think you need these routes here. Create a routes.js file and set them to variables. Import those which needed

Reply:
I'll do that in a separate PR.